docs-src/cloud/prometheus-grafana-setup-cloud.md

```diff
@@ -184,21 +189,21 @@ When you run Prometheus locally, set your target address to port 8077 in your Pr
 
 Example:
```

Comment:
I'm cheating with this suggestion 'cause the issue in question is technically out of scope: 

In the previous line, change `endopint` to `endpoint`.